core/src/main/kotlin/com/justai/jaicf/reactions/Reactions.kt

```diff
@@ -109,7 +109,7 @@ abstract class Reactions {
      *
      * @return [ImageReaction] for logging purposes
      */
-    open fun image(url: String): ImageReaction = createImageReaction(url)
+    open fun image(url: String): ImageReaction = ImageReaction(url, currentState)
```

Comment:
Please, consider either removing the return value or change documentation.
If `Reaction` is needed only for logging purposes and logging happens inside this method then we don't need actually return the `Reaction` object.
The same is true for other reactions.

Reply:
This return value was created to enforce contract between framework and developer, who implements a channel. Using this contract we explicitly tell developer to create our Reaction objects. Default reactions are registered on creation. Following this contract should bring support to logging feature by default. 

This also may look controversial, as all logic is hidden inside `create` method, and developer may wonder why does he need to create this object, but at this point I don't see better solution. 

I'd suggest merging this feature as is with further refactoring possible, if we find a better solution.